mmv1/templates/terraform/resource_definition/identity_platform_project_default_config_deprecation.go.erb

```diff
@@ -0,0 +1,15 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2023 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+DeprecationMessage: "Deprecated. Use the `google_identity_platform_config` resource instead.",
```

Comment:
nit: we may also consider adding the reasons to this deprecation message as well